source/_posts/2017/2017-01-10-jak-psat-zajimave-inzeraty-pro-ajtaky.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: "Jak psát zajímavé inzeráty pro ajťáky"
+perex: '''
+    Už pár let sleduju pracovní PHP trh z pozice programátorů i z pohledu firem. S oběma stranami mám dobré vztah, tuším přibližně co potřebují a co je trápí. Vidím, že se navzájem chtějí, jen komuniace trochu vázne. Rozhodl jsem se to zlepšit a <a href="https://twitter.com/VotrubaT/status/801791233592963077">nabídnul jsem 2 firmám feedback na jejich inzerát</a>. Dnes se mrknenem na inzeráty a jak je dělat líp.
```

Comment:
> Dnes se mrknenem na inzeráty a jak je dělat líp.
Zní divně. Přeformuloval bych to...

Dnes se podíváme na inzeráty a ukážeme si, jak je dělat lépe.